solidity/test/TokenHolder.js

```diff
@@ -1,52 +1,179 @@
 const { accounts, contract } = require('@openzeppelin/test-environment');
 const { expect } = require('../../chai-local');
-const { expectRevert, constants, BN } = require('@openzeppelin/test-helpers');
+const { expectRevert, constants, BN, balance } = require('@openzeppelin/test-helpers');
 
 const { ZERO_ADDRESS } = constants;
 
+const { NATIVE_TOKEN_ADDRESS } = require('./helpers/Constants');
+
 const TokenHolder = contract.fromArtifact('TokenHolder');
 const TestStandardToken = contract.fromArtifact('TestStandardToken');
 
 describe('TokenHolder', () => {
+    const getBalance = async (tokenAddress, account) => {
+        if (tokenAddress === NATIVE_TOKEN_ADDRESS) {
+            return balance.current(account);
+        }
+
+        return (await TestStandardToken.at(tokenAddress)).balanceOf.call(account);
+    };
+
+    const getBalances = async (tokenAddresses, account) => {
+        const balances = {};
+        for (const tokenAddress of tokenAddresses) {
+            balances[tokenAddress] = await getBalance(tokenAddress, account);
+        }
+
+        return balances;
+    };
+
     let holder;
-    let erc20Token;
+    let token;
```

Comment:
An optional suggestion: I'm highly in favor of starting to use token1/token2 (rather than token/token2).

Reply:
Not for now, though